src/watson/registration.py

```diff
@@ -52,6 +52,12 @@ def _resolve_field(self, obj, name):
         """Resolves the content of the given model field."""
         name_parts = name.split("__", 1)
         prefix = name_parts[0]
+        # Check if the attribute exists in the database
+        try:
+            if not getattr(obj, prefix):
```

Comment:
This will throw an AttributeError if the field does not exist on the object, which will break the ability to look up accessors on the SearchAdapter class.

I'm confused about how this would fix your bug. The traceback in your original issue originated from https://github.com/bykof/django-watson/blob/patch-1/src/watson/registration.py#L75

This implies that hasattr(obj, prefix) returns False. If hasattr(obj, prefix) is returning False, then how is getattr(obj, prefix) raising a ObjectDoesNotExist?